.github/workflows/publish.yml

```diff
@@ -0,0 +1,76 @@
+name: Publish package to npmjs
+
+# This workflow runs when code is pushed to `main` (i.e: when a pull request is merged)
+on:
+    push:
+        branches: [main]
+
+# Ensure that only one instance of this workflow executes at a time.
+# If multiple PRs are merged in quick succession, there will only ever be one publish workflow running and one pending.
+concurrency: ${{ github.workflow }}
+
+jobs:
+    version:
+        runs-on: ubuntu-latest
+
+        # OSBotify will update the version on `main`, so this check is important to prevent an infinite loop
+        if: ${{ github.actor != 'OSBotify' }}
+
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                ref: main
+                # The OS_BOTIFY_COMMIT_TOKEN is a personal access token tied to osbotify, which allows him to push to protected branches
+                token: ${{ secrets.OS_BOTIFY_COMMIT_TOKEN }}
+
+            - name: Decrypt & Import OSBotify GPG key
+              run: |
+                cd .github
+                gpg --quiet --batch --yes --decrypt --passphrase="$LARGE_SECRET_PASSPHRASE" --output OSBotify-private-key.asc OSBotify-private-key.asc.gpg
+                gpg --import OSBotify-private-key.asc
+              env:
+                LARGE_SECRET_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
+
+            - name: Set up git for OSBotify
+              run: |
+                git config --global user.signingkey 367811D53E34168C
+                git config --global commit.gpgsign true
+                git config --global user.name OSBotify
+                git config --global user.email infra+osbotify@expensify.com
+            - uses: actions/setup-node@v4
+              with:
+                node-version-file: ".nvmrc"
+                registry-url: "https://registry.npmjs.org"
```

Comment:
Is this `registry-url` necessary? We don't use it in NewDot

Reply:
I think the default is already `https://registry.npmjs.org` so I think we can remove this as it wouldn't really override anything.